openfecwebapp/templates/legal-search-results-murs.html

```diff
@@ -8,7 +8,13 @@
 
 {% block results %}
 {% with murs = results.murs %}
-<div class="message message--info"><p>Results don't include most of the cases closed between 1975 and 1998. Search those cases on <a href="http://www.fec.gov/MUR/">FEC.gov.</a></p></div>
+<div class="message message--info">
+  <h3>This feature is still in progress.</h3>
+  <p>We are actively building the <span className="font-bold">MUR search</span>, and it doesn&#39;t yet include some
+  advanced search functions. If you can&#39;t find what you&#39;re looking for,
+  you can still <a href="http://eqs.fec.gov/eqs/searcheqs">search MURS on the the old fec.gov</a>.</p>
+  <p>Additionally, use the <a href="http://www.fec.gov/MUR/">MUR archive</a> to search for cases closed between 1975 and 1998.</p>
```

Comment:
For the MUR archive link, use `{{ classic_url}}/MUR/`. Right now it will point to fec.gov but eventually will be `classic.fec.gov`.